### PR TITLE
Project model

### LINKED ISSUE #54: Create Project Model in Django App
Include fields for:

- name
- repo url
- project url
- desc
- user id

### PR DESCRIPTION
**Pull request by iandemed**
## Related Issues (include the `#`)
Attempting to close #54 

## Description of Changes Made
Added Project model to developerverse app. New migration file added after testing functionality

## Questions or Comments (optional)
I did not hear back from anyone this morning on whether or not we want a many-to-many relationships between Users and Projects. I opted to keep a one-to-many relationship since that was the assumed default.

### CODE REVIEW

File: back-end/developerverse/models.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
 
 # Create your models here.
+
+class Project(models.Model):
+
+    # We cannot directly use 
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
```
Comment (Underwaterr) [inline back-end/developerverse/models.py]: This comment is a bit confusing. Cannot use what with what?

File: back-end/developerverse/models.py [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+    name = models.CharField(
+        max_length=140
+    )
+    repo_url = models.URLField(
+        max_length=480, blank=True
+    ) # allow for longer URLs
+    project_url = models.URLField(
+        max_length=480, 
+        blank=True
+    )
```
Comment (Underwaterr) [inline back-end/developerverse/models.py]: There's inconsistency in formatting. Are we keeping every property on a new line? 
My recommendation is, unless the line is 80+ characters, keep each property on one line.

File: back-end/developerverse/models.py [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+    desc = models.TextField(
+        max_length=500, 
+        blank=True
+    ) # User can use about 80 words for description
```
Comment (Underwaterr) [inline back-end/developerverse/models.py]: 500 characters feels very short to me. 80 words is reasonable for an average or even above-average length, but that's tough as a hard limit. Let's do 3000 bytes.

**Review by Underwaterr — CHANGES_REQUESTED** [review]

File: back-end/developerverse/models.py (hunk 3 above)
Comment (iandemed) [reply back-end/developerverse/models.py]: I changed it to 3000 characters which I believe should correspond to 3000 bytes. Is the length for the URLs sufficient?

File: back-end/developerverse/models.py (hunk 1 above)
Comment (iandemed) [reply back-end/developerverse/models.py]: I revised the comment to be more specific and point to relevant documentation from Django

**Review by Underwaterr — APPROVED** [review]
Beautiful!

**Pull request merged**